classes/sso/hash/hashing_service.php

```diff
@@ -50,6 +50,20 @@ public static function generate_hash(string $user, \stdClass $mumietask) : mumie
         return $mumieidhash;
     }
 
+    /**
+     * Generate a hash of the userid with a lecturer postfix.
```

Comment:
"Generate a hash of the user id with the lecturer postfix."  is a bit more precise

Reply:
I adjust it to `Generate a hash of the user id with the lecturer postfix`.